sys-boot/raspberrypi-firmware/raspberrypi-firmware-9999.ebuild

```diff
@@ -1,55 +1,39 @@
-# Copyright 1999-2017 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=7
 
-inherit git-r3 readme.gentoo
+inherit mount-boot git-r3 readme.gentoo-r1
 
-DESCRIPTION="Raspberry PI boot loader and firmware"
+DESCRIPTION="Raspberry Pi (all versions) bootloader and GPU firmware"
 HOMEPAGE="https://github.com/raspberrypi/firmware"
-SRC_URI=""
+EGIT_REPO_URI="https://github.com/raspberrypi/firmware"
 
-LICENSE="GPL-2 raspberrypi-videocore-bin"
+LICENSE="GPL-2 BSD"
```

Comment:
This is not correct. Old LICENSE value was correct.
Everything but GPL stuff is exactly under a license known in Gentoo tree as raspberrypi-videocore-bin.

Reply:
I think, this could be a question for the license team, as the ```raspberrypi-videocore-bin``` looks like a default BSD license, that's why I changed it in first place.